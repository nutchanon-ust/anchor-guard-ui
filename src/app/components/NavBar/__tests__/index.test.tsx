import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { NavBar } from '../index';

const shallowRenderer = createRenderer();

describe('<NavBar />', () => {
  it('should match snapshot', () => {
    shallowRenderer.render(<NavBar />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
