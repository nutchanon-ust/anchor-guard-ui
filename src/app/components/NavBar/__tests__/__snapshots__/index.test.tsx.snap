// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should match snapshot 1`] = `
<styled.header>
  <styled.div>
    <Logo />
    <Nav />
  </styled.div>
</styled.header>
`;
